packages/runtime/test/services/NodeBalancer.spec.ts

```diff
@@ -31,7 +32,8 @@ describe('services/LocalGateway', () =>
     {
         it('should throw a node not available error', async () =>
         {
-            const run = async () => emptyBalancer.run('nonExisting', Version.DEFAULT, new Map(), new Map());
+            const request = new Request('nonExisting', Version.DEFAULT, new Map(), new Map());
+            const run = async () => emptyBalancer.run(request);
```

Comment:
I expect this name to be `response` to be consistent with the other tests

Reply:
Same question here as in the LocalGateway comment.